lib/validator/v3/validator_test.go

```diff
@@ -248,6 +248,26 @@ func init() {
 			true,
 		),
 
+		Entry("should accept GlobalNetworkSetSpec with CIDRs and IPs",
```

Comment:
I think we should have a test that a `GlobalNetworkSet` with a label `projectcalico.org/namespace: foo` is rejected, since Global resources are not namespaced.